Efficientdet/base

AgaMiko opened the pull request:
- updated `taco_to_detectwaste` to match new categories (move `crips_packet` from `non_recyclable ` to `metals_and_plastic`)
- renamed detectwaste categories to match new data from epinote -> `non-recyclable` to `non_recyclable`, `metals_and_plastics` to `metals_and_plastic`
- moved all *.py files concering annotations update to `utils/dataset_converter.py`
- added file `annotations_preprocessing.py` that does everything required to have annotations for all sources (taco and epinote) in detectwaste style
- updated `dataset_config.py` with new annotations paths to match `annotations_preprocessing.py`
- deleted unused configs from `dataset_config.py` for better clarity

AnnaBrodecka reviewed (approved):
Everything is looking good.

AgaMiko commented:
- updated `taco_to_detectwaste` to match new categories (move `crips_packet` from `non_recyclable ` to `metals_and_plastic`)
- renamed detectwaste categories to match new data from epinote -> `non-recyclable` to `non_recyclable`, `metals_and_plastics` to `metals_and_plastic`
- moved all *.py files concering annotations update to `utils/dataset_converter.py`
- added file `annotations_preprocessing.py` that does everything required to have annotations for all sources (taco and epinote) in detectwaste style
- updated `dataset_config.py` with new annotations paths to match `annotations_preprocessing.py`
- deleted unused configs from `dataset_config.py` for better clarity
